app/include/zmk/behavior_queue.h

```diff
@@ -10,5 +10,5 @@
 #include <stdint.h>
 #include <zmk/behavior.h>
 
-int zmk_behavior_queue_add(uint32_t position, const struct zmk_behavior_binding behavior,
-                           bool press, uint32_t wait);
+int zmk_behavior_queue_add(struct zmk_behavior_binding_event *event,
```

Comment:
Can probably be `const`:

```suggestion
int zmk_behavior_queue_add(const struct zmk_behavior_binding_event *event,
```